docs/configuration.md

```diff
@@ -38,9 +38,21 @@ These steps should be repeated for each project-version before using any zanata-
 
 You can customize `zanata.xml` with command hooks so that other tools will automatically run before or after Zanata commands. Read about command hooks at the [command hook page on the wiki](https://github.com/zanata/zanata-server/wiki/Client-Command-Hooks).
 
+## Source directory and translation directory
+
+`<src-dir>` and `<trans-dir>` can be defined in zanata.xml. You should use a relative path relative to the directory where zanata.xml lives. This will save you from giving the option on command line each time.
```

Comment:
I think we should flip this so the reason to do it is first:

> To prevent the need to specify the source and translation directories on the command-line, they can be specified in zanata.xml. The source and translation directories are specified in the `<src-dir>` and `<trans-dir>` elements respectively, as shown below. Both paths must be relative to the directory that contains zanata.xml.